bedrock/exp/tests/test_views.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from django.test import override_settings
+from django.test.client import RequestFactory
+
+from mock import patch, ANY
+
+from bedrock.exp import views
+from bedrock.mozorg.tests import TestCase
+
+
+@override_settings(DEV=False)
+@patch('bedrock.exp.views.l10n_utils.render')
+class TestExpFirefoxNew(TestCase):
+    def test_download_template(self, render_mock):
+        req = RequestFactory().get('/exp/firefox/new/')
+        req.locale = 'en-US'
```

Comment:
@pmac Can you help me out with the test failure here please? flak8 is complaining this variable is defined but not used. But if I'm reading it right we're adding a property to a variable that *is* used on the next line. Am I reading that right?

Reply:
Oh I'm looking at the wrong file nevermind >_<